ert_shared/plugins/plugin_manager.py

```diff
@@ -211,6 +227,28 @@ def get_documentation_for_jobs(self):
             )
         return job_docs
 
+    @python3only
+    def get_documentation_for_workflows(self):
+        workflow_config = self.get_ertscript_workflows()
+
+        job_docs = {
+            workflow.name: {
+                "description": workflow.description,
+                "examples": workflow.examples,
+                "config_file": workflow.config_path,
+                "parser": workflow.parser,
+            }
+            for workflow in workflow_config._workflows
```

Comment:
Maybe the `get_workflows` function should be used instead here to avoid accessing the private variable

Reply:
Nvm, I see why it's been done like this!